pkg/provision/buildroot.go

```diff
@@ -151,6 +151,10 @@ WantedBy=multi-user.target
 		return nil, err
 	}
 
+	if err := p.Service("", serviceaction.DaemonReload); err != nil {
```

Comment:
Now that the docker port is hardcoded, I think this can go away. If not, please add a comment describing why we need to reload systemd files.

Reply:
the reason is in kic we install it and it already has a service file

Reply:
This step can take a couple seconds to run on my laptop -- if it's not necessary it would be better to remove it from a performance standpoint